config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -149,7 +149,11 @@ IF(Python_Interpreter_FOUND)
     SET(VIGRANUMPY_INCLUDE_DIRS ${VIGRANUMPY_INCLUDE_DIRS}
         CACHE PATH "include directories needed by VIGRA Python bindings"
         FORCE)
-    SET(VIGRANUMPY_LIBRARIES ${Python_LIBRARIES} ${Boost_PYTHON_LIBRARY})
+    IF(APPLE)
```

Comment:
Maybe we can add a comment pointing to this PR so that other packagers can somewhat understand why we did this.

Reply:
> CONDA_TOOLCHAIN_BUILD

 I didn't know about this environment variable. If this works, lets try it.

I think only the homebrew folks might open an issue, and we can take action once they do.

Ideally, they would contribute an azure pipeline as well, but I don't know homebrew enough to build up the pipeline myself.